internal/operators/ocs/validations.go

```diff
@@ -0,0 +1,120 @@
+package ocs
+
+import (
+	"encoding/json"
+	"math"
+
+	"github.com/docker/go-units"
+	"github.com/openshift/assisted-service/internal/host"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+type OCSValidator struct {
```

Comment:
Consider calling this `Config` and creating another Struct that will implement the logic.
You will need to have logger and maybe other interfaces here like `host.API` that are not part of this struct.
Then it will be easier to make a generic interface out of it.